src/reconciliation/preview.ts

```diff
@@ -0,0 +1,50 @@
+import {LookupQueryResult} from '../lookup/lookup';
+import {RelatedTerm, Term} from '../services/terms';
+import {Literal} from 'rdf-js';
+import {escapeHtml} from '@hapi/hoek';
+
+export function preview(lookupResult: LookupQueryResult) {
+  const term = lookupResult.result;
+  if (term instanceof Term) {
+    return `<html>
+    <head>
+      <meta charset="utf-8" />
+      <style>
+        body { font-family: "Helvetica Neue", Helvetica, Arial, sans-serif; }
+        dt { font-weight: bold; margin-top: 1em; }
+        dd { margin: 0; }
+      </style>
+    </head>
+    <body>
+      <h1>${literal(term.prefLabels)}</h1>
+      <p>${literal(term.scopeNotes)}</p>
+      <dl>
+        ${
+          term.altLabels.length > 0
+            ? `<dt>Alternatieve labels</dt><dd>${literal(term.altLabels)}</dd>`
+            : ''
+        }
+        ${relatedTerms('Bredere termen', term.broaderTerms)}
+        ${relatedTerms('Nauwere termen', term.narrowerTerms)}
+        ${relatedTerms('Gerelateerde termen', term.relatedTerms)}
```

Comment:
Haven’t found a way in the Reconciliation spec to parameterise the language, so decided to use Dutch here. Would English make more sense?

Reply:
Good suggestion! 👍

OpenRefine indeed sends an `Accept-Language` header with preview requests. By inspecting the headers as they arrived at our preview service, I found that `Accept-Language`’s value depends not on the OpenRefine language setting but on the browser’s locale. While not ideal (Dutch users may have their browser set to English), we can respond to the value in our preview service.

Reply:
Excellent!